sniffer.py

```diff
@@ -150,7 +156,23 @@ def main():
                 pkt = result.value[2:2+length]
                 if options.ti_crc:
                     pkt = ti_crc(pkt)
-                pkt = pcap.encode_frame(pkt)
+
+                # metadata format:
+                # 0. RSSI(int8)
+                # 1. Noise Floor(int8)
+                # 2. Flags(uint16)
+                # 3. PHY-specific data struct contains:
```

Comment:
Sorry, I mistyped - I meant "PHY-specific" data.  In any case, my main thought was that timestamp does not seem PHY specific and could be moved to the standard metadata fields.